common/anomalies/hfe_anomaly.txt

```diff
@@ -228,18 +232,11 @@ hfe_anomaly_blight = {
 		modifier = {
 			factor = 0.1
 			exists = from.owner
+			hfe_trigger_hardcore_enabled = yes
```

Comment:
This trigger will make it more likely to happen without hardcore mod.
This line is not nessesary.

Reply:
It SHOULD be more likely, because the point of that Modifier was to make sure the player can also "win" the Great Blight. Without the player involved, there shouldn't be limits on the AI getting it!